src/showcase-merge/app/shared/component-viewer/one-variant-only-notification/one-variant-only-notification.component.ts

```diff
@@ -0,0 +1,24 @@
+import { Component, Input } from '@angular/core';
+
+import { ShowcaseMetaEntry } from '../../meta';
+
+@Component({
+  selector: 'sbb-one-variant-only-notification',
+  templateUrl: './one-variant-only-notification.component.html',
+  styleUrls: ['./one-variant-only-notification.component.scss'],
+})
+export class OneVariantOnlyNotificationComponent {
```

Comment:
I'm aware I'm bikeshedding, but maybe rename this to VariantLimitationComponent? What is your opinion on this?